src/main/scala/uk/gov/nationalarchives/scannotifications/messages/EventMessages.scala

```diff
@@ -19,41 +19,55 @@ object EventMessages {
   case class SlackMessage(blocks: List[SlackBlock])
 
   implicit val scanEventMessages: Messages[ScanEvent] = new Messages[ScanEvent] {
+    private val allowedTags = Set("latest", "intg", "staging", "prod")
```

Comment:
Are we tagging any images with "mgmt"? If so do we need to account for that?

Reply:
We don't currently but we might in the future so it's worth adding now.